src/sentry/tasks/integrations/github/open_pr_comment.py

```diff
@@ -282,6 +282,28 @@ def get_top_5_issues_by_count_for_file(
     )
     project_ids = [p.id for p in projects]
 
+    stackframe_function_name = lambda i: Function(
+        "arrayElement",
+        (Column("exception_frames.function"), i),
+    )
+    multi_if = []
+    for i in range(-STACKFRAME_COUNT, 0):
+        # if, then conditions
+        multi_if.extend(
+            [
+                Function(
+                    "in",
+                    [
+                        stackframe_function_name(i),
+                        function_names,
+                    ],
+                ),
+                stackframe_function_name(i),
+            ]
+        )
+    # else condition
```

Comment:
is this an `else` condition? 🤔 
i might be misreading this lol

looks like we're iterating through, appending a bunch of names given a condition, then adding the last name?

Reply:
the goal is to return the function name that is associated with the stackframe that satisfies the boolean condition. the boolean condition attempts to find a stackframe that contains the function+filename within the top 6 frames in the stacktrace